python/paddle/fluid/tests/unittests/dygraph_to_static/test_basic.py

```diff
@@ -20,6 +20,8 @@
 
 from paddle.fluid.dygraph.jit import dygraph_to_static_graph
 
+from ifelse_simple_func import *
+
```

Comment:
Why put dygraph if/else functions in file `ifelse_simple_func.py` but the corresponding unittests in file `test_basic.py`?  
For more uniform file in directory `./tests/unitests/dygraph_to_static/`, it's better to merge two files.

Reply:
Because the defined  functions in `ifelse_simple_func` may be used in other test files, so I extract these functions into single file.

Moreover, I think the file name `test_basic.py` is not appropriate. The test cases here are all related with `ifelse`. Another PR will be proposed to rename it. The reason why I don't do that because I'm afraid that it may require specified person to approve.

Reply:
> Moreover, I think the file name `test_basic.py` is not appropriate. The test cases here are all related with `ifelse`.

Great. And another file `test_ifelse_basic.py`  can be renamed too  for ease of understanding.

Reply:
OK.